frontend/packages/client/src/hooks/useCommunity.js

```diff
@@ -152,6 +153,7 @@ export default function useCommunity({
               flowAddress.contractAddr
             ),
             contractName: setDefaultValue(contractN, flowAddress.contractName),
+            contractType,
```

Comment:
I think you need to set it here on proposal creation otherwise this will be empty:
```suggestion
            contractType: setDefaultValue(contractN, 'ft'),
```
or you can set the value on the network file so it's not hardcoded here

Reply:
Ah got it - see how that works now. Added value to networkConfig and followed pattern of other fields.